README.md

```diff
@@ -10,16 +10,34 @@ These tutorials will get you up to speed and sending messages with Solace techno
 
 This repository contains code and matching tutorial walk throughs for five different basic Solace messaging patterns. For a nice introduction to the Solace API and associated tutorials, check out the [tutorials home page](https://solacesamples.github.io/solace-samples-dotnet/).
 
+## Obtaining the Solace API
+
+This tutorial depends on you having the Solace Messaging API for C#/.NET (also referred to as SolClient for .NET) downloaded and installed for your project, and the instructions in this tutorial assume you successfully done it. If your environment differs then adjust the build instructions appropriately.
+
+Here are a few easy ways to get this API.
+
+### Get the API: Using nuget.org
+
+Use the NuGet console or the NuGet Visual Studio Extension to download the [SolaceSystems.Solclient.Messaging](http://nuget.org/packages/SolaceSystems.Solclient.Messaging/) package for your solution and to install it for your project.
+
+The package contains the required libraries and brief API documentation. It will automatically copy correct libraries from the package to the target directory at build time, but of course if you compile your program from the command line you would need to refer to the API assemblies and libraries locations explicitly.
+
+Notice that in this case both x64 and x86 API assemblies and libraries have the same names.
+
+### Get the API: Using the Solace Developer Portal
+
+The SolClient for .NET can be [downloaded here]({{ site.links-downloads }}){:target="_top"}. That distribution is a zip file containing the required libraries, detailed API documentation, and examples.
+
```

Comment:
The "Download here" link in the above is a broken link in the readme. That only works in Jekyll.